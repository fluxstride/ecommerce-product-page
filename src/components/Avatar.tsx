import avatar from '../assets/images/image-avatar.png';

const Avatar = () => {
  return (
    <button data-cy='avatar_button' type='button' className='avatar'>
      <img src={avatar} alt='cart' />
    </button>
  );
};

export default Avatar;
