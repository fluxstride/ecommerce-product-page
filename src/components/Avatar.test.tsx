import render from 'react-test-renderer';
import Avatar from './Avatar';

describe('<Avatar/>', () => {
  it('expect the component to match the snapshot', () => {
    const component = render.create(<Avatar />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
