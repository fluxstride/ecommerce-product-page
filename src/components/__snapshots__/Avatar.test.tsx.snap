// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar/> expect the component to match the snapshot 1`] = `
<button
  className="avatar"
  data-cy="avatar_button"
  type="button"
>
  <img
    alt="cart"
    src="image-avatar.png"
  />
  <p>
    tests
  </p>
</button>
`;
